google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageImpl.java

```diff
@@ -572,8 +582,14 @@ public void deleteHmacKey(HmacKeyMetadata hmacKeyMetadata, DeleteHmacKeyOption..
             .setAccessId(hmacKeyMetadata.getAccessId())
             .setProject(hmacKeyMetadata.getProjectId())
             .build();
-    // TODO retries
-    grpcStorageStub.deleteHmacKeyCallable().call(req);
+    Retrying.run(
+        getOptions(),
+        retryAlgorithmManager.getFor(req),
+        () -> {
+          grpcStorageStub.deleteHmacKeyCallable().call(req);
+          return null;
+        },
+        Decoder.identity());
```

Comment:
Instead of identity, could we use an overload which makes Decoder optional?

Reply:
In StorageImpl we're also using `identity()` for the few methods where necessary. These methods are almost exclusively delete methods (which arguably should return something better than void or boolean). So I'm not sure they are enough motivation to overload `Retrying.run`